cosmo_tester/resources/terraform/scripts/osx-cli-test.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+MANAGER_BLUEPRINTS_PATH="/usr/local/opt/cfy/cloudify-manager-blueprints"
+MANAGER_BLUEPRINT_PATH="${MANAGER_BLUEPRINTS_PATH}/simple-manager-blueprint.yaml"
+INPUTS_FILE_PATH="/tmp/bootstrap_inputs.yaml"
+
+CLI_PACKAGE_URL=$1
+PRIVATE_KEY_PATH=$2
+PUBLIC_IP=$3
+PRIVATE_IP=$4
+MANAGER_USER=$5
+
+echo "Installing Cloudify's CLI..."
```

Comment:
I'm going to be putting up a PR soon for the existing CLI packages with an extra line here:
echo "Using CLI package: ${CLI_PACKAGE_URL}"
(extra feedback for Jenkins that's easily greppable for all tests)

It'd be great if you could include that line here.

Reply:
It's in